tests/update_reference_data.sh

```diff
@@ -37,6 +37,7 @@ tests[spe12p]="flow spe1 SPE1CASE2_2P spe1_2p"
 tests[spe1oilgas]="flow spe1 SPE1CASE2_OILGAS spe1_oilgas"
 tests[spe1nowells]="flow spe1 SPE1CASE2_NOWELLS spe1_nowells"
 tests[spe1thermal]="flow spe1 SPE1CASE2_THERMAL spe1_thermal"
+tests[spe1thermal]="flow spe1 SPE1CASE1_FREEBC spe1_freebc"
```

Comment:
unique key please.

Reply:
what do you mean? (I copied the line above and replaced "thermal" by "freebc".)

Reply:
ah, I see. forgot the LHS...

Reply:
should be fixed. it is possibly worth to think about modifying the build system to do this automatically. ( `opm_add_test()` should already get all required information.)

Reply:
i don't think so. this is not related to the build system, nor is it a task for the build system. this is related to jenkins automation only.

Reply:
yes, but the build system could generate the update script. Anyway, this is (mostly) not my cup of tea...

Reply:
Just want to add that not all results are automatically updated. The cell volume calculations need to be manually updated, and that is deliberate.